sdk/src/main/java/com/hedera/hashgraph/sdk/AccountBalanceQuery.java

```diff
@@ -20,12 +19,19 @@
 public final class AccountBalanceQuery extends Query<AccountBalance, AccountBalanceQuery> {
     private final CryptoGetAccountBalanceQuery.Builder builder;
 
+    AccountId accountId;
+    ContractId contractId;
+
     public AccountBalanceQuery() {
         builder = CryptoGetAccountBalanceQuery.newBuilder();
     }
 
     public AccountId getAccountId() {
-        return AccountId.fromProtobuf(builder.getAccountID());
+        if (accountId == null) {
+            return new AccountId(0);
```

Comment:
We should not be returning "ID 0" if the account ID or contract ID has not been set.